pkgs/test_core/lib/src/runner/compiler_pool.dart

```diff
@@ -1,142 +1,57 @@
-// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
```

Comment:
It's a bit ambiguous from github, but if this isn't a new file we may not want to update the copyright.

Reply:
Oh right, this one isn't new.